arm-cdn/2016-10-02/swagger/cdn.json

```diff
@@ -2018,6 +1882,32 @@
       },
       "x-ms-azure-resource": true
     },
+    "GeoFilter": {
+      "description": "Geo filter of a CDN endpoint.",
+      "type": "object",
+      "required": [
+        "relativePath",
+        "action",
+        "countryCodes"
+      ],
+      "properties": {
+        "relativePath": {
+          "description": "Relative path apply geo filter.",
+          "type": "string"
+        },
+        "action": {
+          "description": "Action of the geo filter.",
```

Comment:
What can the customer put over here? Can you update the description with examples or possible values?
This looks like an enum candiate. If so then please provide the valid values as an enum constraint and make sure to use the x-ms-enum extension correctly.